docs/gettinginvolved/infrastructure/release-process.rst

```diff
@@ -34,27 +34,19 @@ Software
 ~~~~~~~~
 
 - A web browser.
-- `gsutil Python package <https://pypi.python.org/pypi/gsutil>`_ on your workstation.
 - `Vagrant`_ (1.6.2 or newer)
 - `VirtualBox`_
-- `virtualenvwrapper`_
 
 .. _`Vagrant`: https://docs.vagrantup.com/
 .. _`VirtualBox`: https://www.virtualbox.org/
-.. _`virtualenvwrapper`: https://virtualenvwrapper.readthedocs.org/en/latest/
 
 Access
 ~~~~~~
 
-- Access to `Google Cloud Storage`_ using `gsutil`_ on your workstation.
-  Set up ``gsutil`` authentication by following the instructions from the following command:
+- Access to `Google Cloud Storage`_.
 
-  .. prompt:: bash $
-
-      gsutil config
-
-- Access to Amazon `S3`_ using `gsutil`_ on your workstation.
-  Set ``aws_access_key_id`` and ``aws_secret_access_key`` in the ``[Credentials]`` section of ``~/.boto``.
+- Access to Amazon `S3`_ with an `Access Key ID and Secret Access Key <https://docs.aws.amazon.com/AWSSimpleQueueService/latest/SQSGettingStartedGuide/AWSCredentials.html>`_.
```

Comment:
This has been the case since we moved our docs from RTD to S3 but kept artifacts on GCS. There is WIP to move artifacts from GCS to S3 so this will be improved soon.